misago/admin/tests/test_generic_admin_list_view.py

```diff
@@ -37,3 +37,12 @@ def test_unicode_is_preserved_in_redirect_querystring(admin_client):
 def test_view_is_not_redirecting_if_flag_is_set_in_querystring(admin_client):
     response = admin_client.get("%s?redirected=1" % list_link)
     assert response.status_code == 200
+
+
+def test_view_session_filter_is_none(admin_client):
```

Comment:
Please make test name more descriptive:

- what state is tested?
- what is expected result?

Look at other tests names in module for inspiration, something like `test_restoring_filters_from_session_handles_filters_entry_being_none`

On sidenote, I am trying to find place where session filters are changed to `None`, but I can't seem to find one 🤔

Reply:
> Look at other tests names in module for inspiration, something like test_restoring_filters_from_session_handles_filters_entry_being_none

OK.

> On sidenote, I am trying to find place where session filters are changed to None, but I can't seem to find one 🤔
```python
            if request.GET.get("set_filters"):
                # Force store filters in session
                session_key = self.filters_session_key
                request.session[session_key] = context["active_filters"] or {}
                refresh_querystring = True
```
You can see my test. My second request has set_filters query params. Whether `set_filters` is `0` or `1`, it's still `str` type. So, the function will be into this part code. 

`context["active_filters"]` is `None` now. The problem is here.